Tests/KlaviyoSwiftTests/APIRequestErrorHandlingTests.swift

```diff
@@ -266,15 +267,15 @@ class APIRequestErrorHandlingTests: XCTestCase {
         initialState.requestsInFlight = [request]
         let store = TestStore(initialState: initialState, reducer: KlaviyoReducer())
 
-        environment.analytics.klaviyoAPI.send = { _, _ in .failure(.rateLimitError(nil)) }
+        environment.klaviyoAPI.send = { _, _ in .failure(.rateLimitError(backOff: 30)) }
```

Comment:
Did you mean to change this? Are you supporting 503 handling here?

Reply:
Sorry, I don't think I'm following. Yes there is logic to handle 503 now, I should probably add a test for that, but not if that's what you are asking?

Reply:
Oh I see your point now. Essentially, you are saying this test should be like the rate limit error tests since we are retrying on 503. I'll look into it.

Reply:
Okay looked into this and turns out this test also covers the 503 case. For both 429 and 503, `environment.klaviyoAPI.send ` returns `KlaviyoAPIError.rateLimitError(backOff: nextBackOffWithJitter)`.